src/Encoder.ts

```diff
@@ -1,9 +1,9 @@
 import { utf8Count, utf8Encode } from "./utils/utf8";
 import { ExtensionCodec, ExtensionCodecType } from "./ExtensionCodec";
-import { setInt64, setUint64 } from "./utils/int";
 import { ensureUint8Array } from "./utils/typedArrays";
 import type { ExtData } from "./ExtData";
 import type { ContextOf } from "./context";
+import { setInt64, setUint64 } from "./utils/int";
```

Comment:
nit: why reorder import? I think some linters complain about normal imports after type-only imports

Reply:
Prettier did that automatically I believe.

Reply:
Oh wait, no. I think it was because I originally removed it and then realised it was actually needed. I'll fix this.